src/platform/windows/events_loop.rs

```diff
@@ -473,7 +473,8 @@ pub unsafe extern "system" fn callback(
                 window_id: SuperWindowId(WindowId(window)),
                 event: Refresh,
             });
-            winuser::DefWindowProcW(window, msg, wparam, lparam)
+            //winuser::DefWindowProcW(window, msg, wparam, lparam) // TODO: Why was this here?
```

Comment:
Like the comment says - DefWindowProc doesn't appear to make much sense for WM_PAINT. In fact it broke the redrawing code in the example. Can we remove this without breaking anything or am I missing something?

This line is also present in the most recent commit from upstream.